From 8fb7f994485d859a79f367c5fb47349a6da535ca Mon Sep 17 00:00:00 2001
From: Sebastian Andrzej Siewior <bigeasy@linutronix.de>
Date: Sat, 26 Oct 2013 00:01:58 +0200
Subject: [PATCH 215/326] lockref: disable 64bit cmpxchg optimization on RT

One of the requirements is that the lock has to fit in an u32 which does
not work with the sleeping locks. We would have to use the bare raw
locks for this and I would like to avoid this.

Signed-off-by: Sebastian Andrzej Siewior <bigeasy@linutronix.de>
---
 lib/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lib/Kconfig b/lib/Kconfig
index b3c8be0da17f..c742a2d89270 100644
--- a/lib/Kconfig
+++ b/lib/Kconfig
@@ -57,6 +57,7 @@ config CMPXCHG_LOCKREF
 	depends on !GENERIC_LOCKBREAK
 	depends on !DEBUG_SPINLOCK
 	depends on !DEBUG_LOCK_ALLOC
+	depends on !PREEMPT_RT_BASE
 
 config CRC_CCITT
 	tristate "CRC-CCITT functions"
-- 
2.0.0

